docs/machine-learning/resources/azure-training-concepts-model-builder.md

```diff
@@ -46,7 +46,7 @@ To create an Azure Machine Learning compute, the following are required:
     Visit the [NC-series Linux VM documentation](/azure/virtual-machines/nc-series?bc=%252fazure%252fvirtual-machines%252flinux%252fbreadcrumb%252ftoc.json&toc=%252fazure%252fvirtual-machines%252flinux%252ftoc.json) for more details on GPU optimized compute types.
 - Compute priority
 
-  - Low-priority: Suited for tasks with shorter execution times. May be impacted by interruptions and lack of availability. Usually costs less because it takes advantage of surplus capacity in Azure.
+  - Low-priority: Suited for tasks with shorter execution times. Can be impacted by interruptions and lack of availability. Usually costs less because it takes advantage of surplus capacity in Azure.
```

Comment:
Minor nit, but these aren't complete sentences. If you change the structure of line 49, make a similar structural change to line 50